centralserver/central/templates/central/base.html

```diff
@@ -158,6 +158,24 @@
               </div>
           </div>
 
+        <div id="closing">
+            <div class="alert alert-warning">
+                <div class="max-width-container">
+                    <div id="announcement-text-container">
+                        <h3>This service is being replaced by <a href="https://learningequality.org/kolibri/" target="_blank">Kolibri</a>.</h3>
+                        <p>
+                            You can export your data until <strong>March 1st 2020</strong>
+                            after which, this service will be taken down.
+                            KA Lite is fully replaced by a more modern, mature, stable and featureful
+                            platform called <em>Kolibri</em>. It is free and Open Source.
+                            By using Kolibri, you will have access to more content, not least newer and more plentyful Khan Academy content.
+                            You will also have online synchronization and data export with
+                            Kolibri Data Hub, to be launched in early 2020.
+                        </p>
```

Comment:
**Important announcement: KA Lite Central Server deprecation** 

The KA Lite Central Server is in the process of being deprecated.
**You have until **March 1st, 2020** to export your data**. After that date, the service will be completely taken down. 
KA Lite is no longer being actively developed, and it's being replaced by [Kolibri](https://learningequality.org/kolibri/), a free and open-source, second-generation platform that contains all of KA Lite videos and exercises, and many, many more content sources and pedagogical tools. You can [test it here](https://kolibridemo.learningequality.org/en/user/#/signin), and [download it here](https://learningequality.org/download/). If you do require a similar central data syncing service, please contact us via info@learningequality.org, so we can best assess your needs and how we can support.

Reply:
@benjaoming Please let me know if you have any comments. Jamie also already reviewed it and okayed it. Thank you!

Reply:
Thanks so much @carinediaz ! I have implemented these changes.

Reply:
Thanks so much @carinediaz ! I have implemented these changes.